### PR TITLE
R4R: Update CLI docs

### PR DESCRIPTION
**Pull request by dgsbl**
WIP: Update md file from asset to params and rename asset.md to token.md

### CODE REVIEW

File: docs/zh/cli-client/oracle.md [hunk 1]
```diff
@@ -42,47 +42,45 @@ iriscli oracle create [flags]
 ### 创建一个新的feed
 
 ```bash
-iriscli oracle create --chain-id=irishub --from=node0 --fee=0.3iris --feed-name="test-feed" --latest-history=10 --service-name="test-service" --input=<request-data> --providers=<provide1_address>,<provider2_address> --service-fee-cap=1iris --timeout=2 --frequency=10 --total=10 --threshold=1 --aggregate-func="avg" --value-json-path="high" --commit
+iris tx oracle create --chain-id=irishub --from=node0 --fee=0.3iris --feed-name="test-feed" --latest-history=10 --service-name="test-service" --input=<request-data> --providers=<provide1_address>,<provider2_address> --service-fee-cap=1iris --timeout=2 --frequency=10 --total=10 --threshold=1 --aggregate-func="avg" --value-json-path="high" --commit
```
Comment (zhiqiang-bianjie) [inline docs/zh/cli-client/oracle.md:45]: flag '--fee' shoule be '--fees'

File: docs/cli-client/token.md [hunk 2]
```diff
@@ -37,25 +38,25 @@ iriscli asset token issue [flags]
...
 ### Send tokens
 
-You can send any tokens you have just like [sending iris](./bank.md#iriscli-bank-send)
+You can send any tokens you have just like [sending iris](./bank.md#iris-tx-bank-send)
 
 #### Send tokens
 
 ```bash
-iriscli bank send --from=<key-name> --to=<address> --amount=10kitty --fee=0.3iris --chain-id=irishub --commit
+iris tx bank send --from=<key-name> --to=<address> --amount=10kitty --fees=0.3iris --chain-id=irishub --commit
```
Comment (zhiqiang-bianjie) [inline docs/cli-client/token.md:51]: iris tx bank [from] [to] [amount]

File: docs/zh/cli-client/bank.md [hunk 3]
```diff
@@ -161,45 +71,6 @@ iriscli bank send --from=<key-name> --to=<address> --amount=<amount> --fee=0.3ir
...
-
-| 名称，速记 | 类型   | 必须 | 默认 | 描述                         |
-| ---------- | ------ | -------- | ---- | ---------------------------- |
-| --amount   | string | Yes      |      | 要销毁的通证数量，例如10iris |
-
-### 销毁通证
-
-```bash
-iriscli bank burn --from=<key-name> --amount=10iris --chain-id=irishub --fee=0.3iris
+iris tx bank send --from=<key-name> --to=<address> --amount=10iris --fees=0.3iris --chain-id=irishub
```
Comment (zhiqiang-bianjie) [inline docs/zh/cli-client/bank.md:74]: iris tx bank send [from_key_or_address] [to_address] [amount] [flags]

File: docs/zh/cli-client/token.md [hunk 4]
```diff
@@ -37,25 +38,25 @@ iriscli asset token issue [flags]
...
 ### 发送通证
 
-您可以像[发送iris](./bank.md#iriscli-bank-send)一样发送任何通证。
+您可以像[发送iris](./bank.md#iris-tx-bank-send)一样发送任何通证。
 
 #### 发送通证
 
 ```bash
-iriscli bank send --from=<key-name> --to=<address> --amount=10kitty --fee=0.3iris --chain-id=irishub --commit
+iris tx bank send --from=<key-name> --to=<address> --amount=10kitty --fees=0.3iris --chain-id=irishub --commit
```
Comment (zhiqiang-bianjie) [inline docs/zh/cli-client/token.md:51]: `iris tx bank send [from_key_or_address] [to_address] [amount] [flags]`

File: docs/cli-client/bank.md [hunk 5]
```diff
@@ -161,45 +69,5 @@ iriscli bank send --from=<key-name> --to=<address> --amount=<amount> --fee=0.3ir
...
-
-| Name, shorthand | Type   | Required | Default | Description                                                |
-| --------------- | ------ | -------- | ------- | ---------------------------------------------------------- |
-| --regexp        | string | true     |         | Regular expression, maximum length 50, e.g. ^[A-Za-z0-9]+$ |
-
-### Set memo regexp for an account address
-
-```bash
-iriscli bank set-memo-regexp --regexp=^[A-Za-z0-9]+$ --from=<key-name> --fee=0.3iris --chain-id=irishub
+iris tx bank send --from=<key-name> --to=<address> --amount=10iris --fees=0.3iris --chain-id=irishub
```
Comment (zhiqiang-bianjie) [inline docs/cli-client/bank.md:72]: `iris tx bank send [from_key_or_address] [to_address] [amount] [flags]`

**Review by chengwenxi — APPROVED** [review]

**Review by SegueII — APPROVED** [review]

**Pull request merged**